spring-boot-project/spring-boot-actuator-autoconfigure/src/main/java/org/springframework/boot/actuate/autoconfigure/tracing/TracingProperties.java

```diff
@@ -53,4 +74,81 @@ public void setProbability(float probability) {
 
 	}
 
+	public static class Baggage {
+
+		/**
+		 * Whether to enable correlation of the baggage context with logging contexts.
+		 */
+		private boolean correlationEnabled = true;
+
+		/**
+		 * List of fields that should be correlated with the logging context. That means
+		 * that these fields would end up as key-value pairs in e.g. MDC.
+		 */
+		private List<String> correlationFields = new ArrayList<>();
```

Comment:
It looks as if there could be a `correlation` group with `enabled` and `fields` properties.

Reply:
Yeah, that's a good idea!